R/metab_simple.R

```diff
@@ -24,7 +24,9 @@
 #'   
 #' @author Alison Appling, Jordan Read; modeled on lakeMetabolizer
 #' @examples
-#'  metab_simple(data=data.frame(date.time=rep(as.Date("2015-04-15"), 24), DO.obs=1:24, DO.sat=rep(14, 24), PAR=sin((1:24)*pi/24)^8, temp.water=15))
+#'  metab_simple(data=data.frame(
```

Comment:
Was thinking about this basic structure. Should we skip the input for `DO.sat` and calculate that internally? Seems a bit redundant w/ `DO.obs`, given we recommend gets passed through `o2.at.sat` anyhow to build `DO.sat`. Also, could support additional args to `o2.at.sat` with the `...`. Just a thought. I see some pros and cons either way.